From 133f3dc515039b24198311840bf5335a1bdd2d78 Mon Sep 17 00:00:00 2001
From: Yannick Fertre <yannick.fertre@foss.st.com>
Date: Wed, 9 Jun 2021 22:50:05 +0200
Subject: [PATCH 1034/1695] dts: stm32mp13: add display support for
 stm32mp135f-dk

Activate support of secure layer on STM32MP13 Disco board.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 463456125bb7642e21b8ce91fa6bc1917124ac1c dts: stm32mp13: add display support for stm32mp135f-dk]
Signed-off-by: Yannick Fertre <yannick.fertre@foss.st.com>
Change-Id: Ib9f25ac1a8e15a417983944c5a0a2c4f50d9f424
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/391786
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 core/arch/arm/dts/stm32mp135f-dk.dts | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp135f-dk.dts b/core/arch/arm/dts/stm32mp135f-dk.dts
index c87108d52..ed1320e6e 100644
--- a/core/arch/arm/dts/stm32mp135f-dk.dts
+++ b/core/arch/arm/dts/stm32mp135f-dk.dts
@@ -380,6 +380,13 @@
 	status = "okay";
 };
 
+&ltdc {
+	pinctrl-0 = <&ltdc_pins_a>;
+	access-controllers-conf-nsec = <&etzpc DECPROT(STM32MP1_ETZPC_LTDC_ID, DECPROT_NS_RW, DECPROT_UNLOCK)>;
+	access-controllers-conf-sec = <&etzpc DECPROT(STM32MP1_ETZPC_LTDC_ID, DECPROT_S_RW, DECPROT_UNLOCK)>;
+	status = "okay";
+};
+
 &lptimer3 {
 	status = "okay";
 
-- 
2.47.2

